engine/Default/planet.inc

```diff
@@ -0,0 +1,25 @@
+<?php
+// Common code for all the planet display pages
+
+// If not on a planet, forward to current_sector.php
+if (!$player->isLandedOnPlanet()) {
+	if (USING_AJAX) {
+		// Auto-click current sector with javascript to avoid issues with ajax
+		// updates when the display page changes.
+		$container = create_container('skeleton.php', 'current_sector.php');
```

Comment:
It's probably worth adding a message to the current sector screen as well saying something about not being on a planet anymore (to help avoid player confusion)